cli/Makefile

```diff
@@ -144,8 +148,13 @@ $(build_bindir)/julia$(EXE): $(EXE_OBJS) $(build_shlibdir)/libjulia.$(SHLIB_EXT)
 $(build_bindir)/julia-debug$(EXE): $(EXE_DOBJS) $(build_shlibdir)/libjulia-debug.$(SHLIB_EXT) | $(build_bindir)
 	@$(call PRINT_LINK, $(CC) $(LOADER_CFLAGS) $(DEBUGFLAGS) $(EXE_DOBJS) -o $@ $(LOADER_LDFLAGS) $(RPATH) -ljulia-debug)
 
+$(BUILDDIR)/julia.expmap: $(SRCDIR)/julia.expmap.in
+	sed <'$<' >'$@' -e 's/@JULIA_SHLIB_SYMBOL_VERSION@/JL_LIBJULIA_$(SOMAJOR)/' \
+		        -e 's/@LLVM_SHLIB_SYMBOL_VERSION@/$(LLVM_SHLIB_SYMBOL_VERSION)/'
```

Comment:
Where is `deps/llvm-ver.make` coming from here? Those version numbers are the wrong values to be used outside of deps, and must instead be computed from the llvm-config-host library.

Reply:
Ah I didn't know about `USE_SYSTEM_LLVM=1`. Yeah agreed, this is wrong when you're using system LLVM

It looks like we also use the wrong shlib name when updating `JL_PRIVATE_LIBS-0`:
https://github.com/JuliaLang/julia/blob/9d839f9794cbe4cc5eb654bcf8064f3f30452705/Makefile#L217-L219